platform/android/export/export.cpp

```diff
@@ -2587,6 +2620,26 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 				EditorNode::get_singleton()->show_warning(TTR("Building of Android project failed, check output for the error.\nAlternatively visit docs.godotengine.org for Android build documentation."));
 				return ERR_CANT_CREATE;
 			}
+
+			if (export_app_bundle) {
+				List<String> copy_args;
+				String output_path;
```

Comment:
Suggest something similar to

String build_type = (p_debug ? "Debug" : "Release");
String build_command = vformat("bundle%s", build_type);